casper/src/test/scala/coop/rchain/casper/genesis/DeployPaymentCostTest.scala

```diff
@@ -287,10 +288,12 @@ object DeployPaymentCostTest {
       tuplespaceHash: ByteString,
       statusChannel: GPrivate
   ): Unit = {
-    val transferStatus = node.runtimeManager.getData(
-      tuplespaceHash,
-      statusChannel
-    )
+    val transferStatus = node.runtimeManager
+      .getData(
+        tuplespaceHash,
+        statusChannel
+      )
+      .unsafeRunSync
```

Comment:
We've had a pinky promise to have a negative balance of `unsafeRunSync` usages in the diff for further pull requests, but at this point I'm not sure if we should require that in tests.

I guess what could be done is to extract the repeated usage in each test to a priv. The ultimate prudence would be to slowly start converting tests into monadic computations. This way they'd show the *actual* way how the tested code is to be used...

Reply:
agreed. Ill figure out how to drop em.